src/main/java/com/archisacadeny/course/CourseRepository.java

```diff
@@ -0,0 +1,53 @@
+package com.archisacadeny.course;
+
+import com.archisacadeny.config.DataBaseConnectorConfig;
+
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.text.MessageFormat;
+
+public class CourseRepository {
+
+    public static void addCourse(String course_name,String course_number,String course_instructor,String enrolled_students){
```

Comment:
we will discuss how to insert data into a table in todays meeting. this is not the right way to do it.